### DISCUSSION TITLE
new file-Jupyter Notebook

### DISCUSSION THREAD

**Discussion by purwto3369** [teamalgoritma/community#144] [Q&A] [answered]
Bagaimana membuat file baru di Jupyter Notebook

**Comment by kevbow** [comment, marked as answer]
Halo @purwto3369, 

Anda dapat menggunakan tombol "New" di sebelah kanan atas lalu pilih environment yang digunakan. Terlampir contoh yang dapat menjadi acuan.
![image](https://user-images.githubusercontent.com/68848168/174622124-547b2e1f-07ce-4273-8ac1-07125a5a65f3.png)
Jangan lupa untuk memindahkan terlebih dahulu path/folder tempat Anda ingin menyimpan file notebook baru tersebut agar file yang dibuat terdokumentasi dengan baik.

Demikian yang dapat saya sampaikan dan semoga bermanfaat, apabila jawaban ini sesuai dengan pertanyaan Anda harap ditandai pada bagian "Mark as answer".
